assignments/homework/sebastian/HW 10-13.py

```diff
@@ -0,0 +1,53 @@
+#HW 10----
+
+%matplotlib inline
+import matplotlib.pyplot as plt
+plt.rc('image', origin='lower')
+plt.rc('figure', figsize=(10, 6))
+
+from astropy.convolution import Gaussian2DKernel
+from astropy.io import fits
+
+gauss = Gaussian2DKernel(3)
+
+ISSA = fits.getdata('data/ISSA_100_LMC.fits')
+plt.imshow(convolve(ISSA, gauss))
```

Comment:
This file starts from scratch, you need to import everything you use again, as well as define variables.

e.g. `convolve`, `gaia_map`, `scipy_convolve` are not defined.